README.md

```diff
@@ -22,3 +21,28 @@ Please check the individual repositories for instructions for building and contr
 
 This project doesn't have 'Issues' enabled. Please add any issues on the individual projects (see above).
 When in doubt, use [xtext-core](https://github.com/eclipse/xtext-core/issues).
+
+## How to Contribute
+
```

Comment:
The oomph stuff is very involving and only interesting for contributors who are into eclipse.
So let's move that description to xtext-eclipse.

For lib, core and extras it is much simpler. I think we don't need to explain anything actually, because we go with the common fork and PR workflow and have a standard gradle build. 
So please lets leverage the simplification we did to the project.